src/Host/Client/Impl/Extensions/RStringExtensions.cs

```diff
@@ -35,7 +46,33 @@ public static class RStringExtensions {
             for (int i = 1; i < s.Length - 1; ++i) {
                 char c = s[i];
                 if (escape) {
-                    sb.Append(c);
+                    // https://stat.ethz.ch/R-manual/R-devel/library/base/html/Quotes.html
+                    switch (c) {
+                        case 'n':
+                            sb.Append("\n");
+                            break;
+                        case 'r':
+                            sb.Append("\r");
+                            break;
+                        case 't':
+                            sb.Append("\t");
+                            break;
+                        case 'b':
+                            sb.Append("\b");
+                            break;
+                        case 'a':
+                            sb.Append("\a");
+                            break;
+                        case 'f':
+                            sb.Append("\f");
+                            break;
+                        case 'v':
+                            sb.Append("\v");
+                            break;
+                        default:
```

Comment:
There's also the hexadecimal escape, e.g. "\x20" for space. Note that those can be single- or double-digit, and they're case-insensitive - `"\xA"` and `"\x0a"` are both valid.